client/src/actions/index.js

```diff
@@ -123,6 +128,7 @@ export const resetPassword = credentials => async (dispatch, getState) => {
 
 export const postEvent = event => (dispatch, getState) => {
   dispatch(triggerRequest(ADD_EVENT));
+  event = humps.decamelizeKeys(event);
```

Comment:
@isaadabbasi shouldnt we do this inside the makeRequest function? Since its same for all requests?